.husky/pre-commit

```diff
@@ -0,0 +1,2 @@
+npm test
+npm run format-check
```

Comment:
could we `npm run format` here to auto-format the files?

Reply:
done, 

![Uploading Screenshot 2025-06-28 at 10.54.29 AM.png…]()